src/rust/shape.rs

```diff
@@ -65,7 +64,7 @@ fn font_coords_to_f2dot14(font: *mut hb_font_t) -> Vec<F2Dot14> {
     };
     coords
         .iter()
-        .map(|&v| F2Dot14::from_bits(v as i16))
+        .map(|&v| NormalizedCoord::from_bits(v as i16))
```

Comment:
If you make this function return `ShaperInstance`, you should be able to pass this iterator directly as the second argument to `ShaperInstance::from_coords` and avoid the collect into an intermediate allocation.

Reply:
Thanks.